src/do_turn.cpp

```diff
@@ -1007,10 +1007,16 @@ bool do_turn()
         if( u.activity.is_interruptible() && u.activity.interruptable_with_kb ) {
             wait_message += string_format( _( "\n%s to interrupt" ), press_x( ACTION_PAUSE ) );
         }
-        wait_refresh_rate = 5_minutes;
+        static const activity_id autodrive_activity_id( "ACT_AUTODRIVE" );
```

Comment:
This should be declared at the global scope (at the beginning of the file).

Reply:
Done.